proposals/raster_to_vector.json

```
@@ -0,0 +1,56 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts single band raster data cube to vector data",
+    "description": "Converts a single band raster data cube into a vector data cube. The bounding polygon of uniform areas of pixels is constructed.",
```

```suggestion
    "description": "Converts a single band raster data cube into a vector data cube. The bounding polygon of uniform areas of pixels is constructed.",
```

I think this may need some more information about the conversion process. Is this a union polygon of all pixel centers? Or really the fully area of all pixel boundaries?